R/learner_survivalmodels_surv_parametric.R

```diff
@@ -128,13 +135,22 @@ LearnerSurvParametric = R6Class("LearnerSurvParametric",
 
     .predict = function(task) {
       pv = self$param_set$get_values(tags = "predict")
-
       newdata = as.data.frame(ordered_features(task, self))
 
+      # use unique train set times
```

Comment:
This is used multiple times. Refactor this into a function

Reply:
Done!